_source/_includes/setup/integrator-cli.md

```diff
@@ -0,0 +1,231 @@
+  {%- elsif include.type == "web" -%}Web Application
+  {%- elsif include.type == "native" -%}Native Application
+  {%- elsif include.type == "service" -%}API Services
+  {%- else -%}{{ include.type | capitalize }} Application
+  {%- endif -%}
+** as the application type, then click **Next**
+5. Enter an app integration name
+  {% if include.type == "spa" %}
+6. In the **Grant type** section, ensure both **Authorization Code** and **Refresh Token** are selected
+7. Configure the redirect URIs:
+   - **Sign-in redirect URIs**: `{% if adoc %}\{% endif %}{{ include.loginRedirectUri }}`
+   {% if include.logoutRedirectUri %}- **Sign-out redirect URIs**: `{% if adoc %}\{% endif %}{{ include.logoutRedirectUri }}`{% else %}- **Sign-out redirect URIs**: `{% if adoc %}\{% endif %}{{ baseUrl }}`{% endif %}
+8. In the **Controlled access** section, select the appropriate access level
+9. Click **Save**
+  {% elsif include.type == "web" %}
+6. Configure the redirect URIs:
+    {% if include.loginRedirectUri and include.logoutRedirectUri %}- **Sign-in redirect URIs**: `{% if adoc %}\{% endif %}{{ include.loginRedirectUri }}`
+   - **Sign-out redirect URIs**: `{% if adoc %}\{% endif %}{{ include.logoutRedirectUri }}`
+    {% elsif include.loginRedirectUri %}- **Sign-in redirect URIs**: `{% if adoc %}\{% endif %}{{ include.loginRedirectUri }}`
+   - **Sign-out redirect URIs**: `{% if adoc %}\{% endif %}{{ baseUrl }}`
+    {% else %}- Accept the default redirect URI values{% if include.framework contains "Spring Boot" %}:
+     - **Sign-in redirect URIs**: `{% if adoc %}\{% endif %}http://localhost:8080/login/oauth2/code/okta`
+     - **Sign-out redirect URIs**: `{% if adoc %}\{% endif %}http://localhost:8080`{% endif %}
+    {% endif %}
+7. In the **Controlled access** section, select the appropriate access level
+8. Click **Save**
+  {% elsif include.type == "native" %}
+6. Configure the redirect URIs:
+    {% if include.loginRedirectUri == include.logoutRedirectUri %}- Use `{{ include.loginRedirectUri }}` for both the **Redirect URI** and **Post Logout Redirect URI**
+    {% else %}- **Redirect URI**: `{{ include.loginRedirectUri }}`
+   - **Post Logout Redirect URI**: `{{ include.logoutRedirectUri }}`
+    {% endif %}
+    {%- if include.loginRedirectUri contains 'com.okta.' -%}
+   (where `{{ include.loginRedirectUri | remove: 'com.okta.' | remove: ':/callback' | remove: '[http://localhost:8100/callback,' | remove: ']' }}.okta.com` is your Okta domain name). Your domain name is reversed to provide a unique scheme to open your app on a device.
+    {% else %}{% endif %}
+7. In the **Controlled access** section, select the appropriate access level
+8. Click **Save**
+  {% elsif include.type == "service" %}
+6. In the **Controlled access** section, select the appropriate access level
+7. Click **Save**
+  {% endif %}
+{% endif %}
+
+{% if include.note %}
+{{ note }}
+{% endif %}
+
+{% capture details %}
+{%- if include.type == "jhipster" -%}
```

Comment:
Let's remove jhipster. We can add it back in if there's a need.

In particular, the instructions to add group claims and to use the default authorization are red flags. Instead of fixing it, let's just take out jhipster as an option